src/pages/decks/index.tsx

```diff
@@ -54,25 +58,43 @@ export default function Home() {
     }
   }
 
-  if (isLoading) {
+  if (loading || (user && isLoading)) {
```

Comment:
isUserLoading and areDecksLoading maybe?

Reply:
Agreed! This was a too-lazy-to-refactor situation. I'll get this cleaned up :)